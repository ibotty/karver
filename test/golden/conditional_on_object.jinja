{% if emptyObjectVar %}
  emptyObjectVar evaluates to true
{% else %}
  emptyObjectVar evaluates to false
{% endif %}

{% if objectVar %}
  objectVar evaluates to true
{% else %}
  objectVar evaluates to false
{% endif %}
