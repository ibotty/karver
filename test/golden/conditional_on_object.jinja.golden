  emptyObjectVar evaluates to false

  objectVar evaluates to true
